exec/java-exec/pom.xml

```diff
@@ -71,10 +71,15 @@
     <!-- </dependency> -->
     <dependency>
       <groupId>org.ow2.asm</groupId>
-      <artifactId>asm-debug-all</artifactId>
-      <version>5.0.3</version>
+      <artifactId>asm-commons</artifactId>
+      <version>6.2.1</version>
       <!-- <scope>test</scope> -->
     </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-util</artifactId>
```

Comment:
I don't see any new dependencies being introduced as part of the PR, why is this necessary? The same for all other newly introduced dependencies.